src/main/scala/ai/privado/exporter/ConsoleExporter.scala

```diff
@@ -25,14 +25,11 @@ object ConsoleExporter {
     val sourceNameIdMap = sources.map((source) => (source.id, source.name)).toMap
 
     // Leakage Number - SourceId Map
-    val leakageSourceMap = dataflowsOutput(Constants.leakages).map(
```

Comment:
Changes in this file are related to formatting changes. It seems @dbMundada didn't run `scalafmt` in last changes he has pushed.